### PR TITLE
Feature/redo breaks computation

### PR DESCRIPTION
**Pull request by krlosMata**
This PR does the following:
- change `breaks-computation` format to include description
- removes old `break-computation` test that works now
- modify `gen-inputs` to skip tx that used over 30M gas

### CODE REVIEW

File: tools/ethereum-tests/gen-inputs.js [hunk 1]
```diff
@@ -210,6 +215,12 @@ describe('Generate inputs executor from ethereum tests GeneralStateTests\n\n', a
                         }
 
                         const currentTest = test[keysTests[y]];
+
+                        // check gas used by the tx is less than 30M
+                        if (Scalar.gt(Scalar.e(currentTest.blocks[0].blockHeader.gasUsed), zkcommonjs.Constants.BATCH_GAS_LIMIT)) {
+                            await updateNoExec(dir, newOutputName, 'tx gas > 30M', noExec);
```
Comment (ignasirv) [inline tools/ethereum-tests/gen-inputs.js:221]: There are a few tests where the tx gas > 30M but can be generated and passed correctly (most of them is not the case).

Comment (krlosMata) [reply tools/ethereum-tests/gen-inputs.js:221]: This parameter is the `gasUsed` by the transaction. If gas used is above 30M will behave differently in our `zkEvmDB` and in when executing them in the `zkevm-rom`

Comment (ignasirv) [reply tools/ethereum-tests/gen-inputs.js:221]: Ohh its true 💯

**Review by laisolizq — APPROVED** [review]
:smile_cat:

**Pull request merged**